### ISSUE TITLE
About VLMs confidence

### ISSUE THREAD

**Issue by LebinLiang** [sinatayebati/vlm-uncertainty#8]
Thank you very much for your work. My project requires access to the confidence scores output by the VLMs used (such as Qwen-VL or VILA). I would like to ask whether it is possible to modify your work to enable me to quickly implement this functionality?

**Comment by sinatayebati** [comment]
@LebinLiang Of course! my integration already covers that and returns the logits with which you can either compute log probs or probs of the model prediction.
